src/url.php

```diff
@@ -320,6 +321,32 @@ function qtranxf_get_url_info( string $url ): array {
     return $urlinfo;
 }
 
+/**
+ * Returns the base admin url of the WordPress backend name e.g. wp-admin
+ *
+ * @author Sebastian Poetter https://github.com/poetter-sebastian
+ * @link https://github.com/qtranslate/qtranslate-xt/pull/1324 repo pull request
```

Comment:
We don't want to track the authors by function, this is an open-source project.
Also the link is unnecessary.